src/core/org/apache/jmeter/resources/messages_de.properties

```diff
@@ -536,3 +536,5 @@ warning=Warnung\!
 web_server_domain=Server Name oder IP\:
 web_testing_retrieve_images=Hole alle Bilder und Java Applets (nur HTML Dateien)
 you_must_enter_a_valid_number=Sie m\u00FCssen ein g\u00FCltige Nummer eingeben
+font.decrease=Decrease font to %d pt
```

Comment:
This is not a german sentence :)
If you are not sure, leave it out, or use `Verkleinere Font auf %d pt` (The next one would be `Vergrößere Font auf %d pt` - the `ö` and `ß` would have to be translated to \u... sequences, still.)